src/applications/edu-benefits/sass/edu-benefits.scss

```diff
@@ -28,3 +28,77 @@
 .row .edu-intro-spacing {
   margin-bottom: 7rem;
 }
+
+// autocomplete component
+.ds-c-autocomplete__list {
```

Comment:
Have a todo to rename these styles- for now I just used what was in CMS design system.